harness/setup.py

```diff
@@ -1,5 +1,11 @@
 from setuptools import find_packages, setup
 
+packages = find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"])
+client_pkg = find_packages(where="determined/swagger")
+
+for pkg in client_pkg:
+    packages.append("determined.swagger." + pkg)
```

Comment:
> I don't see any of the following details in this PR, which I think are necessary for supporting this "copy bindings into the module" in a production-quality way:
> 
> * a target in bindings/Makefile that actually copies the generated python files into place.
> * a `check` target in bindings/Makefile which forces the above copy to occur and ensures that there are no updated files (see `make -C master check-gen`)
> * the `check` target should also ensure that the generated requirements for the swagger bindings are also present in the python setup.py file.  Probably you will want to offload that into a helper script.
> * that `check` target should be called as part of the circleci linting actions; i.e. nobody should be allowed to change the swagger definitions without also committing the resulting swagger generated files into place.

Added a script for diffing the two directories and printing out files that need to be updated. This will make sure the definitions and generated files are always in-sync.

Swagger requirements are already added in setup.py. Will add a task for the helper script.